packages/vue/src/components/molecules/SfGallery/SfGallery.vue

```diff
@@ -86,19 +90,15 @@ export default {
     },
     imageWidth: {
       type: Number,
-      default: 422,
```

Comment:
what do you think about not removing default values and adding null values instead? There are default values or required values for the other components so I think it's good to keep consistent

Reply:
I did that thanks for noticing :)